server/sonar-server/src/main/java/org/sonar/server/property/ws/IndexAction.java

```diff
@@ -64,7 +64,7 @@
   private static final String COMMA_ENCODED_VALUE = "%2C";
 
   public static final String PARAM_KEY = "key";
-  private static final String PARAM_COMPONENT = "resource";
+  public static final String PARAM_COMPONENT = "resource";
 
   private final DbClient dbClient;
   private final ComponentFinder componentFinder;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "componentFinder" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1068)